Updated gender references in Cinema Event dialogues

LuximGato opened the pull request:
*Added Pronoun Swaps for Events_Main_140_120_C, Events_Main_140_180_C, Events_Main_140_230_C, Events_Main_140_250_C,
Events_Main_140_260_C,
Events_Main_140_280_C,
Events_Main_140_310_C,
Events_Main_140_320_C, and Events_Main_140_330_C

List of Edits made: Event_Main_140_070_C Changed line 31 him -> her
Event_Main_140_120_C Changed line 72 guy -> girl
Event_Main_140_180_C cutting line 36 short
Event_Main_140_230_C line 8 removed -kun added -chan
Event_Main_140_250_C line 86 him-> her
Event_Main_140_260_C line 133 removed "boys"
Event_Main_140_280_C line 27 removed "You boys are cute.", line 31 removed "You boys don't understand women at all."
Event_Main_140_310_C line 131 "Hey baby"  removed baby
Event_Main_140_320_C line 34 he -> she
Event_Main_140_330_C line 57 -kun to -chan, line 58 him -> her, line 61 his -> her

LuximGato commented:
Added Pronoun Swaps for Event_Cmmu_002 (Yukari's Social Link Events), .msg file in raw files, and etc

List of Edits: 
Event_Cmmu_002_010_C line 42 -kun to -chan
Event_Cmmu_002_020_C line 52 -kun to -chan
Event_Cmmu_002_030_C line 72 -kun to chan
Event_Cmmu_002_040_C line 87 -kun to chan
Event_Cmmu_002_055_C line 28 boyfriend -> girlfriend, line 77 you're a girl -> we're both girls, line 89 men -> others, line 104 removed so what if im a girl, line 105 removed behind men
Event_Cmmu_002_060_C line 56 -kun to -chan
Event_Cmmu_002_070_C line 2 -kun to chan, line 49 -kun to chan
Event_Cmmu_002_080_C line 7 -kun to chan, line 95 -kun to chan
Event_Cmmu_002_090_C line 28 -kun to chan, line 73 -kun to chan, line 84 -kun to chan, line 121 -kun to chan
Event_Cmmu_002_190_C line 28 -kun to chan
Event_Cmmu_002_320_C line 3 -kun to chan
Event_Cmmu_002_340_C line 54 -kun to chan
Event_Cmmu_002_360_C line 19 -kun to chan
Event_Cmmu_002_370_C line 16 -kun to chan, line 36 removed "for boys"
Event_Cmmu_002_570_C line 45 -kun to chan, line 58 boyfriend -> girlfriend, line 64, kun -> chan boyfriend -> girlfriend, line 126 -kun to chan, line 136 -kun to chan
Event_Cmmu_002_650_C line 2 -kun to chan
Event_Cmmu_002_660_C line 22 -kun to chan
Event_Cmmu_002_665_C line 22 -kun to chan, line 102 -kun to chan
Event_Cmmu_002_715_C line 29 -kun to chan, line 49 -kun to chan, line 58 -kun to chan
Event_Cmmu_002_717_C line 13 -kun to chan, line 22 -kun to chan
Event_Cmmu_002_735_C line 4 -kun to chan, line 12 boy -> girl, line 30 kun to chan line 70 -kun to chan, line 79 -kun to chan,
Event_Cmmu_002_737_C line 24 -kun to chan, line 34 -kun to chan,
Event_Cmmu_002_740_C line 34 -kun to chan,

MadMax1960 commented:
Please undo the line removals for these. PR's specifically for now should just be specific to gender changes. 

Ie your "Boyfriend" to "Girlfriend" line is fine, but someone might disagree with your other line changes. To prevent this we should finish changing all needed lines pronouns then work on fine tuning things like that.

LuximGato commented:
Removed Event_Main_140_180_C, Event_Main_140_260_C, Event_Main_140_280_C, Event_Main_140_310_C

Updated: 
Event_Cmmu_002_055_C line 28 boyfriend -> girlfriend line 89 men -> others
Event_Cmmu_002_370_C line 16 -kun to -chan 

Added Event_Qest_610_155_C, Event_Qest_650_020_C, Event_Qest_650_050_C
Event_Qest_610_155_C line 71 gentlemen -> lady, his -> her
Event_Qest_650_020_C line 13 -kun to -san
Event_Qest_650_050_C line 9 -kun to -san

-san is used for teachers referring to female students*
